doc/releases/migration-guide-4.1.rst

```diff
@@ -186,6 +186,7 @@ Entropy
 Ethernet
 ========
 
+* Deprecated eth_mcux driver was removed.
```

Comment:
(:github:`84825`)

Reply:
This hasn't been addressed yet @decsny .